client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -14,7 +14,7 @@ import config from 'config';
 import Button from 'components/button';
 import { cancelAndRefundPurchase, cancelPurchase, submitSurvey } from 'lib/upgrades/actions';
 import { clearPurchases } from 'state/purchases/actions';
-import { isHappychatAvailable, hasActiveHappychatSession } from 'state/happychat/selectors';
+import { isHappychatAvailable, hasActiveHappychatSession } from 'extensions/happychat/state/selectors';
```

Comment:
Same here, if we aren't able to remove the direct references to the extension, we should at least comment prominently that it needs to be resolved.